symbology/apply-unique-values-with-alternate-symbols/src/main/java/com/esri/samples/apply_unique_values_with_alternate_symbols/ApplyUniqueValuesWithAlternateSymbolsSample.java

```diff
@@ -0,0 +1,231 @@
+import com.esri.arcgisruntime.mapping.view.AnimationCurve;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.symbology.MultilayerPointSymbol;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.symbology.SymbolReferenceProperties;
+import com.esri.arcgisruntime.symbology.UniqueValueRenderer;
+
+public class ApplyUniqueValuesWithAlternateSymbolsSample extends Application{
+
+  private MapView mapView;
+
+  private FeatureLayer featureLayer;
+
+  private final static String FEATURE_SERVICE_URL = "https://sampleserver6.arcgisonline.com/arcgis/rest/services/SF311/FeatureServer/0";
+
+  private Button btnReset;
+
+  private Label lblScale;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Apply unique values with alternate symbols");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access base maps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a point located at San Francisco, CA to be used as the viewpoint for the map
+      Point vPoint = new Point(-13631205.660131, 4546829.846004, SpatialReferences.getWebMercator());
+
+      // create a map with the standard imagery basemap style
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // create a map view and set the map to it
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // set a viewpoint to the map view centered on a point
+      mapView.setViewpointCenterAsync(vPoint, 25000.0);
+
+      // creating box, button, and label UI components
+      VBox vBox = controlsVBox();
+
+      // resetting map viewpoint on button press
+      btnReset.setOnAction(event -> {
+        // reset the map viewpoint
+        mapView.setViewpointAsync(new Viewpoint(vPoint, 25000.0), 1.5f, AnimationCurve.EASE_IN_OUT_SINE);
+      });
+
+      // updating label scale values
+      mapView.addViewpointChangedListener(event->
+        // update label text
+        lblScale.setText("Current scale: 1:" + String.format("%.2f", mapView.getMapScale())));
+
+      // creating simple markers where the blue square and yellow diamond are alternate symbols
```

Comment:
just another tense update here, and some others to address following this one
```suggestion
      // create simple marker symboles where the blue square and yellow diamond are alternate symbols
```